frontend-react/e2e/pages/authenticated/admin/receiver-status.ts

```diff
@@ -1,13 +1,15 @@
-import { expect, Locator } from "@playwright/test";
-import { endOfDay, format, startOfDay, subDays } from "date-fns";
-import { RSReceiverStatus } from "../../../../src/hooks/api/UseReceiversConnectionStatus/UseReceiversConnectionStatus";
+import {expect, Locator} from "@playwright/test";
```

Comment:
what happened to formatting here? `import {expect, Locator}` vs `import { expect, Locator }`.

also, don't we want to import the playwright methods from our base test? 

```
import { test as baseTest, expect } from "../../test";
```

Reply:
We don't import the methods from this file. We import the methods from the spec files.